x-pack/test/performance/services/index.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { services as functionalServices } from '../../functional/services';
+import { PlaywrightService } from './playwright';
+
+export const services = {
+  es: functionalServices.es,
+  kibanaServer: functionalServices.kibanaServer,
+  esArchiver: functionalServices.esArchiver,
+  retry: functionalServices.retry,
+  playwright: PlaywrightService,
```

Comment:
I think we should rename this service to something like "performanceTesting", it does much more than just manage Playwright at this point.